docs/bokeh/source/docs/first_steps/examples/first_steps_2_add_circles.py

```diff
@@ -9,10 +9,13 @@
 # create a new plot with a title and axis labels
 p = figure(title="Multiple glyphs example", x_axis_label="x", y_axis_label="y")
 
+# set the background color for the plot
+p.background_fill_color = "#fafafa"
+
```

Comment:
As Bryan has pointed out in the issue, I would try to keep this example as simple as possible (since it is only the second chapter of the first steps guide). I'd leave out the `background_fill_color` and stay with the default. I don't believe this would impact contrast much!

Reply:
Sounds good, thanks for the feedback @tcmetzger

Reply:
FWIW changing the background was my suggestion here, I like the plain white background much less as more time goes by, but unfortunately the default we are stuck with for now.